tests/post_training/pipelines/gpt.py

```diff
@@ -86,7 +87,7 @@ def preprocess_function(examples):
             dataset_config_name="sst2",
             preprocess_function=preprocess_function,
             num_samples=num_samples,
-            dataset_split="validation",
+            dataset_split="train",
```

Comment:
Why was it "validation" before? Why is it "train" now?

Reply:
Something wrong with dataset, in result OV model falls on collection statistics.
Now it works, there is no validation for this model, it's no affect on something else.

Reply:
> Something wrong with dataset, in result OV model falls on collection statistics.

Does it fail right now on develop too? Could you please share a error log?

Reply:
revert to validation dataset
post_training_quantization_performance/69 -pass